### PR TITLE
Updated Validation Attributes

### PR DESCRIPTION
**Pull request by Cjoseph4**
the hasSuccess, hasDanger, hasWarning attributes have been consolidated into one key:value attribute. #524

### CODE REVIEW

**Comment by Cjoseph4** [comment]
This is #524

**Comment by tabersays** [comment]
@Cjoseph4 We use the branch and pull method. You have done the fork and pull. It seems to have messed with our CI. We need you to change over this method.

File: src/bossy-ui/components/form-label/form-label.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div *ngIf="config?.text" [ngClass]="{'has-success': config?.hasSuccess, 'has-warning': config?.hasWarning, 'has-danger': config?.hasDanger}">
+<div *ngIf="config?.text" [ngClass]="{'has-success': config?.hasValidation, 'has-warning': config?.hasValidation, 'has-danger': config?.hasValidation}">
```
Comment (tabersays) [inline src/bossy-ui/components/form-label/form-label.html]: This logic states that if the config has the variable hasValidation and it is truthy add the classes has-warning has-success and has-danger. We are not going to want all of them at once.

File: src/bossy-ui/components/form/form.html [hunk 2]
```diff
@@ -2,26 +2,26 @@
 	<div *ngFor="let element of config.elements" class="form-group">
 		<bossy-form-label [config]="element.label"></bossy-form-label>
 		<div [ngSwitch]="element.type">
-			<div *ngSwitchCase="'number'" [ngClass]="{'has-success': element.hasSuccess, 'has-warning': element.hasWarning, 'has-danger': element.hasDanger}">
+			<div *ngSwitchCase="'number'" [ngClass]="{'has-success': element.hasValidation, 'has-warning': element.hasValidation, 'has-danger': element.hasValidation}">
```
Comment (tabersays) [inline src/bossy-ui/components/form/form.html]: This logic states that if the config has the variable hasValidation and it is truthy add the classes has-warning has-success and has-danger. We are not going to want all of them at once. And repeat this message for all below.

**Comment by glawson** [comment]
@Cjoseph4 - closing - do it right :)

**Pull request closed without merging**